ehb_service/conf/base.py

```diff
@@ -39,14 +39,15 @@
 # although not all choices may be available on all operating systems.
 # If running in a Windows environment this must be set to the same as your
 # system time zone.
-TIME_ZONE = None
+from tzlocal import get_localzone
+TIME_ZONE = get_localzone().zone
 
 SITE_ID = 1
 
 # If you set this to False, Django will make some optimizations so as not
 # to load the internationalization machinery.
 USE_I18N = False
-USE_ETAGS = True
+# USE_ETAGS = True
```

Comment:
Is this a real comment?

Reply:
removed: 70b94dc0577eeb5e2c319a0ef740a793e126e937